tests/nodes/supporting_nodes/test_user.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+import cript
+from cript.nodes.exceptions import UneditableAttributeError
+
+
+def test_user_from_json():
+    """
+    tests just to see if a user node can be created from json
+    """
+    pass
+
```

Comment:
we can implement that

Reply:
I tried writing this test and the deserialization is coming out wrong. There is an extra field called ` "groups": []`

any ideas, what is going wrong?


---

**Original user json:**
```json
{
    "node": "User",
    "username": "my username",
    "email": "user@email.com",
    "orcid": "0000-0000-0000-0002"
}
```

---

**Output user Json:**

```json
{
    "node": "User",
    "username": "my username",
    "email": "user@email.com",
    "orcid": "0000-0000-0000-0002",
    "groups": []
}
```

Reply:
The output is correct. The user node has a "groups" field. And here it is an empty list.